src/disco/metrics/metrics.xml

```diff
@@ -213,6 +213,8 @@ metric introduced.
     <counter name="TransactionSanitizeFailure" summary="Number of transactions that failed to sanitize." />
     <counter name="TransactionNotExecutedFailure" summary="Number of transactions that did not execute. This is different than transactions which fail to execute, which make it onto the chain." />
 
+    <counter name="PrecompileVerifyFailed" summary="Number of transactions that failed precompile verification and thus will not execute." />
```

Comment:
nit: maybe keep naming consistent with the above ones, `PrecompileVerifyFailure`

Reply:
Good call. I renamed it.